src/lib/main.js

```diff
@@ -172,7 +169,7 @@ module.exports = async (argv, env) => {
 
   for (let action of actions) {
     info(`Starting action: ${action}…`);
-    await executeAction(action, `${instanceUrl.href}medic`, extraArgs);
+    await executeAction(action, instanceUrl.href, extraArgs);
```

Comment:
I think there are actions that extract the server URL from instanceURL at runtime. 
`Ctrl+f`-ing for `.replace(/\/medic$` yields 3 actions: `create-users`, `upload-sms-from-csv` and `upload-custom-translation`, there could be more that parse the instanceURL differently.